CHANGELOG.md

```diff
@@ -1,9 +1,43 @@
 # Changelog
 
-## unreleased
-
-* [ENHANCEMENT] TSDB: Make the wlog watcher read segments synchronously when not tailing. #13224
-* [BUGFIX] Agent: Participate in notify calls. #13223
+## 2.49.0-rc.0 / 2023-12-12
+
+* [FEATURE] Promtool: Add `--run` flag promtool test rules command. #12206
+* [FEATURE] SD: Add support for `NS` records to DNS SD. #13219
+* [FEATURE] UI: Add heatmap visualization setting in the Graph tab, useful histograms. #13096
+* [FEATURE] Scraping: Add `scrape_config.enable_compression` (default true) to disable gzip compression when scraping the target. #13166
+* [FEATURE] PromQL: Add a `promql-experimental-functions` feature flag containing some new experimental PromQL functions. #13103 NOTE: More experimental functions might be added behind the same feature flag in the future. Added functions:
+  * Experimental `mad_over_time` (median absolute deviation around the median) function. #13059
+  * Experimental `sort_by_label` and `sort_by_label_desc` functions allowing sorting returned series by labels. #11299
+* [FEATURE] SD: Add `__meta_linode_gpus` label to Linode SD. #13097
+* [FEATURE] Add the ability to track staleness when an explicit timestamp is provided by scrape target. Controlled by `scrap_config.track_timestamps_staleness` field. Useful for e.g. cAdvisor scraping. #13060
```

Comment:
This was in v2.48.0.

```suggestion
```

Reply:
Thanks, missed https://github.com/prometheus/prometheus/pull/13066 -- fixed!